### ISSUE TITLE
LanguageService & StringFormat for GroupItem template

### ISSUE THREAD

**Issue by lipchev** [Catel/Catel#1156]
### Steps to reproduce
Could not create a valid reproduction outside of the project, so it could very well be something I'm doing wrong and/or there is something that I'm not carrying over to the repro. 
Anyway here is the example snippet that is failing for me

1. Created an GroupItem template style with a localized string formatting for the number of children:
<Style x:Key="ExpandableContentGroupStyle" TargetType="{x:Type GroupItem}">
        <Setter Property="Template">
            <Setter.Value>
                <ControlTemplate TargetType="{x:Type GroupItem}">
                    <Expander>
                        <Expander.Header>
                            <StackPanel Orientation="Horizontal">
                                <ContentPresenter Content="{Binding Path=Name}" />
                                
                                <TextBlock
                                    Text="{Binding Path=ItemCount, StringFormat={x:Static properties:Resources.ItemsGroupCountFormat}}"/>
                                
                                <TextBlock
                                    Text="{catel:LanguageBinding ResourceName=ProductComments}">
                                
                                <TextBlock
                                    Text="{Binding Path=ItemCount, Mode=OneWay, StringFormat={catel:LanguageBinding ResourceName=ItemsGroupCountFormat}}"/>
                            </StackPanel>
                        </Expander.Header>
                        <ItemsPresenter Margin="15,0,0,0" />
                    </Expander>
                </ControlTemplate>
            </Setter.Value>
        </Setter>
    </Style>
2. The style is placed in an external ResourceDictionary (merged in App.xaml)
3. The group style is used with a ListView with 
            <ListView.GroupStyle>
                <GroupStyle ContainerStyle="{StaticResource ExpandableContentGroupStyle}" />
4. Language service is registered (resources from multiple assemblies): value for the ItemsGroupCountFormat is like:
<data name="ItemsGroupCountFormat" xml:space="preserve">
    <value>({0} items)</value>
  </data>
5. For the vanilla test the DefaultCulture (en) is used, but otherwise culture is set as expected, failback culture set as well (also played around with the caching/preloading options but to no effect). 

Platform:
.NET version:
4.5.2, 
Catel 5.3.0 beta (just updated, previous test was 5.0- but I doubt it matters)

### Expected behaviour

TextBlocks 1 and 3 should display the same value (correctly formatted to the ItemsGroupCountFormat shown above)
TextBlock 2 should also display the correctly localized string (no formatting there, just for testing)

### Actual behaviour

TextBlocks 1 and 2 display the expected strings

TextBlock 3 displays the value(number) as if no string format had been set

All string formatting outside of this scenario seem to be working fine and so does changing to another culture (even for the DataTemplates /Styles in the external resources). 

I'm sorry I can't bring a workable reproduction to the issue but there it is- if you have any ideas as to why this is could be failing- any help would be greatly appreciated.

**Comment by GeertvanHorrik** [comment]
I don't think you can use an updatable markup extension to set the StringFormat. Not sure if it's bindable, I don't use this very often. I usually use converters for this kind of stuff.

Note that TB2 uses a static value, where the LanguageBinding will return a dynamic value.
